nuxeo-admin-console-web/angular-app/src/app/features/bulk-action-monitoring/services/bulk-action-monitoring.service.ts

```diff
@@ -0,0 +1,18 @@
+import { REST_END_POINTS } from "./../../../shared/constants/rest-end-ponts.constants";
+import { NetworkService } from "./../../../shared/services/network.service";
+import { Injectable } from "@angular/core";
+import { BehaviorSubject, Observable } from "rxjs";
+import { BulkActionMonitoringInfo } from "../bulk-action-monitoring.interface";
+@Injectable({
+  providedIn: "root",
+})
+export class BulkActionMonitoringService {
+  pageTitle: BehaviorSubject<string> = new BehaviorSubject("");
+  constructor(private networkService: NetworkService) {}
+  performBulkActionMonitoring(id: string | null): Observable<BulkActionMonitoringInfo> {
+    return this.networkService.makeHttpRequest<BulkActionMonitoringInfo>(
+      REST_END_POINTS.BULK_ACTION_MONITORING,
+      { urlParam: id }
```

Comment:
There should be an object inside urlParam for generic purpose
{urlParam: {id: id}}
OR in short
{urlParam: {id}}

Reply:
There could be more type of params added in the url for a generic url, and not only id